File: Exercicios_guanaraba/Com_Dart/bin/ex012_Calculando_Desconto.dart
//Faça u algoritmo que leia o preço de um produto e mostre seu novo preço, com 5%de desconto

import 'dart:io';

void main(){

  print('Insira o valor do produto: ');
  double valor = double.parse(stdin.readLineSync());
  double cdesc = valor - (valor*5/100);
  print ('O produto com 5% de desconto fica : $cdesc');



}

File: Exercicios_guanaraba/Com_Ruby/exercicios/ex012_Calculando_Desconto.rb
#Faça um algoritmo que leia o preço de um produto e mostre seu novo preço, com 5% de desconto

puts 'Insira o valor do produto: '
valor = gets.chomp.to_f
cdesc = valor - (valor * 5/100)
puts "O produto com 5% de desconto fica: #{cdesc}"